### ISSUE TITLE
How to find what is a valid menu placement

### ISSUE THREAD

**Issue by SamuelCharpentier** [Adobe-CEP/CEP-Resources#422]
I can't find the documentation on how to build a placement attribute value for a menu element.

Any help is welcome!

**Comment by ErinFinnegan** [comment]
Hi @SamuelCharpentier ! Are you asking about Photoshop, another app, or is this a cross-application use case?

**Comment by SamuelCharpentier** [comment]
Hi @ErinFinnegan, I’m asking about any documentation that would help me figure out how to create such attribute in any supported application as I’m working on a universal CEP Manifest builder (for any app supporting it) that’s a little more advanced than the one offered in the CEP bundle core project. I want to be able to help the user create this attribute with easy access to hints, integrated validation and links to official documentation. The issue is that there seems to be no official documentation on the matter.

Do you think you could help me find such documentation?

**Comment by SamuelCharpentier** [comment]
Yes I've seen the new UXP announced. I'm hoping I could eventually port some of what I've developed over to it when the time comes. As of now, I'm still just learning software development as a hobby.

I've seen most of these and there still seems to be a missing piece to the documentation. [In the CookBook, more specifically in the section related to the menu tag](https://github.com/Adobe-CEP/CEP-Resources/blob/master/CEP_11.x/Documentation/CEP%2011.1%20HTML%20Extension%20Cookbook.md#customize-extension-menu), it says: 

> You can customize the extension menu by editing `<menu/>` item in manifest. Here is an example. In this example, the Adobe Add-ons extension is displayed under Windows main menu, rather than extensions menu under Windows. You can customize the location of extension to somewhere else by changing the value of attribute **Placement** in `<menu/>` item.

And then follows an example with `<Menu Placement="'Main:&amp;Window',600.0,'KBSCE Window menu'">Adobe Add-ons</Menu> ` in it. The issue is that there is no explanation of what `"'Main:&amp;Window',600.0,'KBSCE Window menu'"` means, where it came from or how to edit it or make it different.
